cmd/fetchd/cmd/gen_asi_upgrade_manifest.go

```diff
@@ -64,30 +64,33 @@ type ASIUpgradeSupply struct {
 }
 
 type ContractValueUpdate struct {
-	Address string `json:"address"`
-	From    string `json:"from"`
-	To      string `json:"to"`
+	Address     string      `json:"address"`
+	ValueUpdate ValueUpdate `json:"value_update"`
```

Comment:
When I'm looking a this right now, I would prefer to revert this.
I know, I was the one who asked for the change, but now when I'm thinking about it, it just adds unnecessary stuff & sub-structure in to the resulting manifest json file.